### PR TITLE
code cleanup & EditorConfig

### PR DESCRIPTION
**Pull request by dguittet**
We've discussed coding style guidelines and generally agreed that something such as google's c++ guide would be worth trying out.

Using an `.editorconfig` file with EditorConfig plugin allows reformatting existing code and automatic consistency of new code.

### CODE REVIEW

File: include/lk/absyn.h [hunk 1]
```diff
@@ -63,48 +65,59 @@ typedef std::equal_to<std::string> lk_string_equal;
 
 #define lk_tr(s) lk::get_translation(s)
 
-namespace lk
-{
-	lk_string get_translation(const lk_string &);
-	void set_translation_function(lk_string(*f)(const lk_string&));
+namespace lk {
+    lk_string get_translation(const lk_string &);
+
```
Comment (daniel-thom) [inline include/lk/absyn.h]: How did these extra new lines get added?

Comment (dguittet) [reply include/lk/absyn.h]: What do you mean-- as in why is it double spaced?

Comment (daniel-thom) [reply include/lk/absyn.h]: Yes.

**Comment by sjanzou** [comment]
All ssc tests and all SAM default configurations unchanged and files open and save in VS2019 without issue

**Pull request merged**